08-rails/00-unit-testing-models-specs.md

```diff
@@ -41,63 +41,28 @@ This means when Rake or Rails runs the test task, it will use the minitest-rails
 
 Then run `bundle install`
 
-#### Installing Minitest Spec
+#### Step 2:  Installing Minitest Spec
 
-Then we will ask Rails to modify test_helper.rb to use spec style describe and it blocks with `rails generate minitest:install`  It will ask you if you want to overwrite the test_\_helper.rb file.  Answer yes (Y).
+Then we will ask Rails to modify test_helper.rb to use spec style describe and it blocks with `rails generate minitest:install`  It will ask you if you want to overwrite the test_\_helper.rb file.  Answer yes **(Y)**.
```

Comment:
The "test_\\_helper.rb" bit at the end of the second sentence is giving us two underscores. I think we only need "test_helper.rb" there.